### ISSUE TITLE
The model testing accuracy is very low

### ISSUE THREAD

**Issue by aonrdbit** [xbfu/PyTorch-ParameterServer#1]
Hello, have you tested the accuracy of the model under this PS structure? 

When I used cifar10 dataset to train the resnet18 model, the accuracy  of trainset was normal, but the accuracy of the testset was about 10%, that is to say, the model had no effect. 

This is because model.eval() is used during testing.

 But I don't know why. Can you help me explain why? I would appreciate it if I could receive your reply.
